doc/doxygen/Makefile

```diff
@@ -1,7 +1,7 @@
 RIOTBASE=$(shell git rev-parse --show-toplevel)
 # Generate list of quoted absolute include paths. Evaluated in riot.doxyfile.
 export STRIP_FROM_INC_PATH_LIST=$(shell \
-    git ls-tree -dr --full-tree --name-only HEAD |\
+    git ls-tree -dr --full-tree --name-only HEAD core drivers sys |\
```

Comment:
I didn't know pkg was supposed to be documented by the main Doxygen build. I'm actually more inclined to exclude the pkg tree altogether and let each pkg provide its own documentation.